src/injected/styles/injected.scss

```diff
@@ -62,7 +63,7 @@
         width: 25%;
         max-width: 25%;
         user-select: text;
-        min-width: 550px;
+        min-width: 520px;
```

Comment:
Is this an intentional regression of the change in #584?

Reply:
Yes, intentional; the other css changes include tightening the padding of the button contents such that the increased width is no longer necessary